cypress/e2e/pages/photoGalleryPage/tests.js

```diff
@@ -6,7 +6,7 @@ export const testsThatFollowSmokeTestConfig = ({
   variant,
 }) => {
   describe(`testsThatFollowSmokeTestConfig to run for ${service} ${variant} ${pageType}`, () => {
-    it('should render a description for the page', () => {
+    it.skip('should render a description for the page', () => {
```

Comment:
This scenario could (should/might already) be covered by integration tests therefore could be deleted